src/Compilers/CSharp/Portable/Symbols/Symbol.cs

```diff
@@ -600,6 +600,22 @@ public override bool Equals(object obj)
         }
 
         public bool Equals(ISymbol other)
```

Comment:
>public bool Equals(ISymbol other) [](start = 8, length = 33)

Are there any callers to this overload? If not, please remove.

Reply:
Yes this is still on the public interface due to `IEquatable<ISymbol>`

---
In reply to: [305994519](https://github.com/dotnet/roslyn/pull/37247#discussion_r305994519) [](ancestors = 305994519)

Reply:
As we discussed offline, let's make these two overloads explicit implementations if possible.

---
In reply to: [305996642](https://github.com/dotnet/roslyn/pull/37247#discussion_r305996642) [](ancestors = 305996642,305994519)